java-plugin/pom.xml

```diff
@@ -160,25 +160,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
                 <executions>
-                    <execution>
-                        <id>copy-bundle</id>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>copy</goal>
-                        </goals>
-                        <configuration>
-                            <artifactItems>
-                                <artifactItem>
-                                    <groupId>${project.groupId}</groupId>
-                                    <artifactId>${project.artifactId}</artifactId>
-                                    <version>${project.version}</version>
-                                    <type>jar</type>
-                                    <overWrite>true</overWrite>
-                                </artifactItem>
-                            </artifactItems>
-                            <outputDirectory>../lib</outputDirectory>
```

Comment:
Recently, the construction of the Docker image has been modified to be based on the files in the "lib" folder. So, it is necessary either to adapt the Dockerfile or to keep this configuration. I didn't spot any other impacts in the project but maybe I missed something @dedece35?
https://github.com/green-code-initiative/ecoCode/blob/9257205cfc72f5415be28c6ead0f2954f613c4f1/Dockerfile#L9

Reply:
I don't think it's a good practice to put this particularity in the POM.
It is better to adapt the Dockerfile to take into account the necessary JARs where they were built by default.
What I just did, following your remark, in the commit: c889050c4d2f9a87f1bf1055b45d0785e13e4f92